Makefile

```diff
@@ -43,4 +43,15 @@ deploy-createx-l2:
 	forge script script/DeployCustomCreatex.s.sol --rpc-url $L2_RPC_URL --private-key $PRIVATE_KEY --slow --no-metadata
 
 check-configs: 
-	bun lzConfigCheck.cjs
\ No newline at end of file
+	bun lzConfigCheck.cjs
+
+chain1 := $(shell cast chain-id -r $(L1_RPC_URL))
+chain2 := $(shell cast chain-id -r $(L2_RPC_URL))
+symbol := $(shell cat deployment-config/$(fileL1) | jq -r ".boringVault.boringVaultSymbol")
+post-deploy:
+	mkdir -p ./nucleus-deployments/$(symbol)
+	cp ./broadcast/deployAll.s.sol/$(chain1)/run-latest.json ./nucleus-deployments/$(symbol)/L1.json
+	cp ./broadcast/deployAll.s.sol/$(chain2)/run-latest.json ./nucleus-deployments/$(symbol)/L2.json
+	cp ./deployment-config/$(fileL1) ./nucleus-deployments/$(symbol)/L1Config.json
+	cp ./deployment-config/$(fileL2) ./nucleus-deployments/$(symbol)/L2Config.json
+	cd nucleus-deployments && git checkout -b $(symbol) && git add . && git commit -m "$(symbol) deployment" && git push origin $(symbol) && git checkout main
```

Comment:
Deployments repo should have branch protection on `main` that prevents a direct push and this should automatically push a PR if possible